### PR TITLE
Fix issue #338, TG-153 #closed, implement new progression design

### PR DESCRIPTION
**Pull request by schalkneethling**
@alexgibson r?

### CODE REVIEW

File: data/js/lib/token-manager.js [hunk 1]
```diff
@@ -21,20 +21,15 @@
     // listens for a tokens event from the add-on and enables tokens
     // that has been received by the user.
     addon.port.on('tokens', function(tokens) {
+        let activeImgPath = '/data/media/icons/progress/star_active.svg';
```
Comment (alexgibson) [inline data/js/lib/token-manager.js]: If you know this path is never going to change may as well use `const`?

File: data/js/utility/content2.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 (function() {
     // retrieve button from the page
     var button = document.querySelector('button');
-    var nextButton = document.querySelector('.next-button');
-    var nextToken = document.querySelector('#token3');
+    var nextButton = document.getElementById('next-button');
+    var nextToken = document.querySelector('#token2');
```
Comment (alexgibson) [inline data/js/utility/content2.js]: `getElementById`

File: data/media/icons/progress/star1.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 720 720"><title>Group 6</title><path fill="#E2E2E2" d="M711.2 304.7L554.4 456.4l36 216v7.7c0 10.3-5.1 20.6-18 20.6-5.1 0-12.9-2.6-18-5.1L361.5 592.7 168.6 695.6c-5.1 2.6-10.3 5.1-18 5.1-12.9 0-18-10.3-18-20.6v-7.7l36-216L11.8 304.7c-5.1-5.1-10.3-12.9-10.3-20.6 0-12.9 12.9-18 23.1-20.6l216-30.9 97.7-195.4c5.1-7.7 10.3-18 20.6-18s18 10.3 20.6 18l97.7 195.4 216 30.9c10.3 2.6 23.1 7.7 23.1 20.6 5.2 7.8.1 15.5-5.1 20.6z"/><path fill="#FFF" d="M410.4 513h-54V312.5l-18 18-28.3 23.1-25.7-33.4 82.3-66.9h43.7V513z"/></svg>
```
Comment (alexgibson) [inline data/media/icons/progress/star1.svg]: New line at end of files here and below?

Comment (schalkneethling) [inline data/media/icons/progress/star1.svg]: I used svgomg to optimize so, perhaps it is ok?

File: data/tmpl/utility/content5.html [hunk 4]
```diff
@@ -23,23 +23,26 @@
...
+            <input type="image" src="/data/media/icons/progress/star5.svg" width="30" height="30" alt="Token for completing step 5" class="token-trigger" data-step="5" data-track="utility" />
+          </li>
+          <li id="trophy">
+            <img src="/data/media/icons/progress/trophy.svg" width="30" height="30" alt="Trophy image" />
           </li>
         </ul>
-        <h2 class="token-awarded hidden" aria-hidden="true">Here’s your badge!</h2>
-        <button id="prize" class="button small hidden" aria-hidden="true">Claim your prize!</button>
+        <h2 class="token-awarded hidden" aria-hidden="true">You’ve got 5 stars!</h2>
+        <button id="prize" class="button small hidden" aria-hidden="true">Claim your prize now!</button>
```
Comment (alexgibson) [inline data/tmpl/utility/content5.html]: `type="button"`

File: data/media/icons/progress/star1.svg (hunk 3 above)
Comment (alexgibson) [inline data/media/icons/progress/star1.svg]: I don't think svgomg should make a difference?

Comment (schalkneethling) [inline data/media/icons/progress/star1.svg]: I was referring to specifically the the empty new line but, I have added those.

**Comment by alexgibson** [comment]
Aside from the comments above, code-wise this looks good. r+wc 💸

**Comment by schalkneethling** [comment]
Thanks @alexgibson 🚴  - I have addressed all of those, merging.

**Pull request merged**